### ISSUE TITLE
StaticAccess - ability to add exceptions for specific class names

### ISSUE THREAD

**Issue by gregrobson** [phpmd/phpmd#145]
I'm getting lots of false positives for Laravel development as they have designed the architecture to hide the objects and shorten the syntax.

Could a list of classes (if possible with namespace support, although in this case I don't need it) be added as exceptions to this rule so that I could exclude this special classes such as _Input, Cache, Database, Mail, Paginator_ etc

``` php
$value = Cache::get('key');
// is actually this behind the scenes
$value = $app->make('cache')->get('key');
```

More details on the facades used in Laravel.
http://laravel.com/docs/facades#practical-usage

Apart from that this tool is awesome :thumbsup:
